Gopkg.toml

```diff
@@ -282,3 +290,7 @@ required = [
 [[constraint]]
   name = "github.com/instana/go-sensor"
   version = "1.4.12"
+
+[[constraint]]
+  name = "github.com/ExpediaDotCom/haystack-client-go"
+  revision = "3ee3ba12ce07ae900855f7cf3e3f543418e865bf"
```

Comment:
why are you using a revision instead of a version? (https://github.com/ExpediaDotCom/haystack-client-go/releases/tag/0.2.3)

Reply:
Fixed... the commit I needed was available before the 0.2.3 was released at the time of this PR.